Client/Frontend/Browser/URLBarView.swift

```diff
@@ -342,6 +346,8 @@ class URLBarView: UIView {
                 }, completion: { _ in
                     // remove the clone and setup the actual tab button
                     newTabsButton.removeFromSuperview()
+                    self.clonedTabsButton = nil
```

Comment:
Alternatively, you could make `clonedTabsButton` weak if you wanted so we wouldn't have to explicitly be set to nil (it would be cleared after removing it from its superview).